Twitch.Net/TwitchApi.Streams.cs

```diff
@@ -15,12 +15,13 @@ public partial class TwitchApi
         /// <summary>
         /// Get streams for specific games
         /// </summary>
-        /// <param name="gameIds">Array of game ids. Limit: 100</param>
+        /// <param name="gameIds">Array of game ids. Limit: 10</param>
         /// <param name="first">Amount of streams. Limit: 100</param>
+        /// <param name="language">Stream language. Limit: 100</param>
         /// <param name="after">Cursor for pagination</param>
         /// <param name="before">Cursor for pagination</param>
         /// <returns><see cref="HelixPaginatedResponse{HelixStream}"/> with streams</returns>
-        public async Task<HelixPaginatedResponse<HelixStream>> GetStreams(string[] gameIds, int first = 20, string after = null, string before = null)
+        public async Task<HelixPaginatedResponse<HelixStream>> GetStreams(string[] gameIds, int first = 20, string language = null, string after = null, string before = null)
```

Comment:
Wenn es bei den Sprachen ein Limit von `100` gibt, sollte der Parameter in der Signatur auch als ```string[]``` definiert sein und `languages` heißen.

Außerdem müsste die Methoden umbenannt werden, da die Signatur identisch mit denen der anderen Methoden ist. (z.B. `GetStreamsWithGameIds`)